test/runtests.jl

```diff
@@ -33,16 +33,21 @@ function run_nest(text::String; opts = Options(), style = DefaultStyle())
     JuliaFormatter.nest!(style, t, s)
     t, s
 end
-run_nest(text::String, max_margin::Int) =
-    run_nest(text, opts = Options(max_margin = max_margin))
+run_nest(text::String, margin::Int) =
+    run_nest(text, opts = Options(margin = margin))
 
-yasfmt1(s, i, m; kwargs...) = fmt1(s; kwargs..., i = i, m = m, style = YASStyle())
-yasfmt(s, i, m; kwargs...) = fmt(s; kwargs..., i = i, m = m, style = YASStyle())
+function yasfmt(s, i, m; kwargs...)
+    # use DefaultStyle options to make writing tests easier
+    kws = merge(options(DefaultStyle()), kwargs)
+    return fmt(s; kws..., i = i, m = m, style = YASStyle())
```

Comment:
oh i think this isn't actually overwriting YAS options in the way i intended, and that's why tests fail for YAS -- will fix